doc/sphinx/introduction.rst

```diff
@@ -30,33 +30,30 @@ Coq has an interactive mode in which commands are interpreted as the
 user types them in from the keyboard and a compiled mode where commands
 are processed from a file.
 
--  The interactive mode may be used as a debugging mode in which the
-   user can develop his theories and proofs step by step, backtracking
-   if needed and so on. The interactive mode is run with the `coqtop` 
-   command from the operating system (which we shall assume to be some
-   variety of UNIX in the rest of this document).
+-  The interactive mode may be used as a debugging mode in which the users can
+   develop their theories and proofs step by step, backtracking if needed and
+   so on. The interactive mode is generally run with the help of an IDE, such
+   as CoqIDE, documented in :ref:`coqintegrateddevelopmentenvironment`,
+   Emacs with Proof-General (available at https://proofgeneral.github.io/),
+   or jsCoq to run Coq in your browser (see https://github.com/ejgallego/jscoq).
+   The `coqtop` read-eval-print-loop can also be used directly.
```

Comment:
While I'd love to see `coqtop` assigned the role of a debugging utility, as you did previously/